apps/opik-frontend/src/plugins/comet/utils.ts

```diff
@@ -12,3 +12,18 @@ export const buildUrl = (
 
   return `${BASE_URL}${path}${FROM_PARAM}${workspaceNameParameter}${search}`;
 };
+
+export const loadScript = (url: string) => {
```

Comment:
this is so we dont fail in case its not on comet.com ?

Reply:
This will be run only for Comet integration, it is a part of Comet plugin.